garden_ai/backend_client.py

```diff
@@ -50,3 +50,14 @@ def update_doi_on_datacite(self, payload: dict):
     def publish_garden_metadata(self, garden: PublishedGarden):
         payload = json.loads(garden.json())
         self._post("/garden-search-record", payload)
+
+    def upload_garden_metadata(
```

Comment:
should this be named `upload_notebook` or something instead? or are you envisioning this method eventually doing something with garden metadata, too?

Reply:
Oof, no this is a mistake. Probably some copypasta. I'll make it something like upload_notebook, thanks for catching